tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-ortmodule-test-clear-cache-pipeline.yml

```diff
@@ -0,0 +1,25 @@
+trigger: none
+
+jobs:
+- job: Onnxruntime_Linux_GPU_ORTTraining_Clear_Cache
```

Comment:
what is this pipeline? do we need this?

Reply:
Our CI caches some datasets and model snapshots. This pipeline cleans the cache from time to time to prevent bloating the VMs

Besides preventing failures on CI due to missing data, some dataset download add significant to build time. Using a local cache is a good advantage, but we need to prune it from time to time

Reply:
does this pipeline run on each build agent?

---
In reply to: [591913480](https://github.com/microsoft/onnxruntime/pull/6973#discussion_r591913480) [](ancestors = 591913480)